FirebaseMLModelDownloader/Tests/Unit/ModelDownloaderTests.swift

```diff
@@ -87,23 +56,51 @@ final class ModelDownloaderTests: XCTestCase {
     XCTAssertEqual(modelInfoRetriever.modelInfo?.path, nil)
   }
 
+  /// Unit test to download model info - makes an actual network call.
   func testDownloadModelInfo() {
     let testApp = FirebaseApp.app()!
-    let functionName = #function
-    let testModelName = "\(functionName)-test-model"
+    let testModelName = "image-classification"
     let modelInfoRetriever = ModelInfoRetriever(
       app: testApp,
-      projectID: Constants.Options.projectID,
       modelName: testModelName
     )
     let expectation = self.expectation(description: "Wait for model info to download.")
     modelInfoRetriever.downloadModelInfo(completion: { error in
-      guard let downloadError = error else { return }
-      XCTAssertEqual(downloadError, .notFound)
-      print("ERROR: Model not found on server.")
+      XCTAssertNil(error)
+      XCTAssertNotNil(modelInfoRetriever.modelInfo)
```

Comment:
nit: You may use [XCTUnwrap](https://developer.apple.com/documentation/xctest/3380195-xctunwrap) instead like e.g. [here](https://github.com/firebase/firebase-ios-sdk/blob/f198987a1aa147a265248d0558722d3f1868624d/FirebaseCore/Tests/SwiftUnit/FirebaseAppTests.swift#L48) to test for `nil` and avoid extra unwraps below. Don't forget to add `throws` to the test method declaration like [here](https://github.com/firebase/firebase-ios-sdk/blob/f198987a1aa147a265248d0558722d3f1868624d/FirebaseCore/Tests/SwiftUnit/FirebaseAppTests.swift#L38).

Reply:
I couldn't get `XCTUnwrap` to work with the completion handler, so I've used `guard let` instead. However, the error handling in general is worth revisiting in the future.